pset-functions.js

```diff
@@ -208,6 +306,25 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+function doubleLetters(word) {
+   //Converted word to individual letters array
+   const splitWord = word.split('')
+   //Created false variable as finalCheck kept displaying undefined
+   const notFound = false
+   //Checks each letter to see if it matches the consecutive element
+   for ( i = 0; i < splitWord.length; i++ ) {
+      if(splitWord[i] === splitWord[i + 1]) {
+         return true
+      }
+   }
+   //Returns false if for loop produces no results
+   return notFound
+}
+
+const finalCheck = doubleLetters('apple')
+
+console.log(finalCheck)
+
```

Comment:
@cswebdesign Good work on this problem here. Good work looping through each letter and checking for that repeating value with the conditional you have here. One suggestion you can remove the variable` notFound` and you can just `return false` on line 321
Take a look at the solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)